src/by/oop/task1/File.java

```diff
@@ -0,0 +1,46 @@
+package by.oop.task1;
+
+public class File {
+    private String name = "";
+    private Directory directory;
+    private String content = "";
+
+    public File(String name, Directory directory, String content) {
+        this.name = name;
+        this.directory = directory;
+        this.content = content;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Directory getDirectory() {
+        return directory;
+    }
+
+    public String getContent() {
+        return content;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void setDirectory(Directory directory) {
```

Comment:
not sure, this method is required. It violates immutability, i.e. if not required then remove it